docs/root/configuration/subscription_stats.rst

```diff
@@ -0,0 +1,19 @@
+.. _subscription_statistics:
+
+Subscription statistics
+=======================
+
+The following statistics are generated for all subscriptions.
```

Comment:
Can you clarify what a "subscription" is and maybe cross link to relevant xDS/APIv2 docs? Maybe the title of this should be xDS subscription? WDYT? (Basically I think to the casual reader it won't be clear what this page is about)

Reply:
Sure. Will do.